<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="28300" started-at="2020-06-16T08:56:52Z" finished-at="2020-06-16T08:57:21Z">
    <groups>
    </groups>
    <test name="chrome" duration-ms="28300" started-at="2020-06-16T08:56:52Z" finished-at="2020-06-16T08:57:21Z">
      <class name="ResponsiveLayout.VTigerLoginPage">
        <test-method status="PASS" signature="base(java.lang.String)[pri:0, instance:ResponsiveLayout.VTigerLoginPage@6121c9d6]" name="base" is-config="true" duration-ms="18964" started-at="2020-06-16T08:56:52Z" finished-at="2020-06-16T08:57:11Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- base -->
        <test-method status="FAIL" signature="Login()[pri:0, instance:ResponsiveLayout.VTigerLoginPage@6121c9d6]" name="Login" duration-ms="6820" started-at="2020-06-16T08:57:11Z" finished-at="2020-06-16T08:57:18Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Layout Test Failed :  expected [0] but found [11]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Layout Test Failed :  expected [0] but found [11]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at ResponsiveLayout.VTigerLoginPage.Login(VTigerLoginPage.java:71)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:ResponsiveLayout.VTigerLoginPage@6121c9d6]" name="tearDown" is-config="true" duration-ms="2431" started-at="2020-06-16T08:57:18Z" finished-at="2020-06-16T08:57:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- ResponsiveLayout.VTigerLoginPage -->
    </test> <!-- chrome -->
  </suite> <!-- Suite -->
</testng-results>
